packages/components/nodes/llms/Azure OpenAI/AzureOpenAI.ts

```diff
@@ -0,0 +1,157 @@
+                type: 'password'
+            },
+            {
+                label: 'Model Name',
+                name: 'modelName',
+                type: 'options',
+                options: [
+                    {
+                        label: 'text-davinci-003',
+                        name: 'text-davinci-003'
+                    },
+                    {
+                        label: 'ada',
+                        name: 'ada'
+                    },
+                    {
+                        label: 'text-ada-001',
+                        name: 'text-ada-001'
+                    },
+                    {
+                        label: 'babbage',
+                        name: 'babbage'
+                    },
+                    {
+                        label: 'text-babbage-001',
+                        name: 'text-babbage-001'
+                    },
+                    {
+                        label: 'curie',
+                        name: 'curie'
+                    },
+                    {
+                        label: 'text-curie-001',
+                        name: 'text-curie-001'
+                    },
+                    {
+                        label: 'davinci',
+                        name: 'davinci'
+                    },
+                    {
+                        label: 'text-davinci-001',
+                        name: 'text-davinci-001'
+                    },
+                    {
+                        label: 'text-davinci-002',
+                        name: 'text-davinci-002'
+                    },
+                    {
+                        label: 'text-davinci-fine-tune-002',
+                        name: 'text-davinci-fine-tune-002'
+                    },
+                    {
+                        label: 'gpt-35-turbo',
+                        name: 'gpt-35-turbo'
+                    }
+                ],
+                default: 'text-davinci-003',
+                optional: true
+            },
+            {
+                label: 'Temperature',
+                name: 'temperature',
+                type: 'number',
+                default: 0.9,
+                optional: true
+            },
+            {
+                label: 'Azure OpenAI Api Instance Name',
+                name: 'azureOpenAIApiInstanceName',
+                type: 'string',
+                placeholder: 'YOUR-INSTANCE-NAME'
+            },
+            {
+                label: 'Azure OpenAI Api Deployment Name',
+                name: 'azureOpenAIApiDeploymentName',
+                type: 'string',
+                placeholder: 'YOUR-DEPLOYMENT-NAME'
+            },
+            {
+                label: 'Azure OpenAI Api Version',
+                name: 'azureOpenAIApiVersion',
+                type: 'string',
+                placeholder: 'YOUR-API-VERSION'
+            },
+            {
+                label: 'Azure OpenAI Api Embeddings Deployment Name',
```

Comment:
Embeddings needed? 
<img width="713" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/2fca3423-a6d2-4a05-92fc-97bca2ed9f22">

Reply:
`azureOpenAIApiEmbeddingsDeploymentName` & `azureOpenAIApiCompletionsDeploymentName` is created under the `AzureOpenAIInput` interface, so I listed it additional parameter.